tls/utils.py

```diff
@@ -24,3 +26,44 @@ def _decode(self, obj, context):
 
 def UBInt24(name):  # noqa
     return _UBInt24(construct.Bytes(name, 3))
+
+
+class BytesAdapter(construct.Adapter):
+
+    def _encode(self, obj, context):
+        if not isinstance(obj, bytes):
+            raise construct.AdaptationError("{} requires bytes, got {}".format(
+                self.subcon.name, repr(obj)))
+        return obj
+
+    def _decode(self, obj, context):
+        return obj
+
+
+def PrefixedBytes(name, length_field=construct.UBInt8("length")):  # noqa
```

Comment:
Can you add a docstring here, describing this, and noting the default length field?